### PR TITLE
fix(icon): replaced Clubhouse logo 

### PR DESCRIPTION
**Pull request by robinrendle**
Switching out the Clubhouse logo because of their rebranding. Now it’s called Shortcut. This change affects the following:

#### 1. The Clubhouse and Shortcut integrations

CleanShot 2021-09-17 at 16 23 40
CleanShot 2021-09-17 at 16 23 34

#### 2. Linked Issues icon in the issue details page
CleanShot 2021-09-17 at 16 09 00

### CODE REVIEW

**Review by Jesse-Box — COMMENTED** [review]
Was the clubhouse logo used anywhere in the app?

**Comment by ceorourke** [comment]
@Jesse-Box it's an issue creation integration so a small b&w version is used on the issue details page so the user can create or view an already created Shortcut/Clubhouse issue. `static/app/icons/iconShortcut.tsx` adds that and it's exported in `static/app/icons/index.tsx`. @robinrendle could you post an image of that as well?

File: src/sentry/static/sentry/images/logos/logo-shortcut.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (leeandher) [inline src/sentry/static/sentry/images/logos/logo-shortcut.svg:3]: @robinrendle  I'm not sure how SVGs work but @ceorourke's point was valid.  I just faked some react props to test it, do you know how to change the SVG to work like the old one?

image

**Review by leeandher — CHANGES_REQUESTED** [review]
The icon doesn't seem to be rendering properly for the linked issue list, is there something different about the SVGs?

File: src/sentry/static/sentry/images/logos/logo-shortcut.svg (hunk 1 above)
Comment (robinrendle) [reply src/sentry/static/sentry/images/logos/logo-shortcut.svg]: Okay – fixed. I figured out the problem with the SVG I was using.

**Review by leeandher — APPROVED** [review]
👏 looks great, thanks!

File: static/app/plugins/components/pluginIcon.tsx [hunk 2]
```diff
@@ -102,6 +102,7 @@ export const ICON_PATHS = {
   rocketchat,
   rookout,
   segment,
+  shortcut: clubhouse,
```
Comment (EvanPurkhiser) [inline static/app/plugins/components/pluginIcon.tsx:105]: Can we rename the file too?

Comment (robinrendle) [reply static/app/plugins/components/pluginIcon.tsx:105]: That’s what I tried at first but it broke the icon and wouldn’t display. Spent a bunch of time trying to figure out why that’s the case but I was thoroughly stumped.

**Pull request merged**